testing/test_terminal.py

```diff
@@ -649,7 +649,8 @@ def test_more_quiet_reporting(self, testdir):
         assert "===" not in s
         assert "passed" not in s
 
-    def test_report_collectionfinish_hook(self, testdir):
+    @pytest.mark.parametrize("param", ["", "--collect-only"])
```

Comment:
I think you can `"param", [(), ("--collect-only",)] ` and then you don't need the ternary below

Reply:
I did this initially, but then the parametrized node id becomes "param0" and "param1", that's why I took this approach. Still want me to change it?

Reply:
@nicoddemus you can still pass ids in using ids or param(..., id)

Reply:
duh of course, will change.

Reply:
Fixed, thanks guys. 👍